lib/services/internalEngine/index.js

```diff
@@ -624,4 +570,51 @@ class InternalEngine {
   }
 }
 
+function indexName (index, collection, { internal=true } = {}) {
+  const prefix = internal
+    ? '%'
+    : '#';
+
+  return `${prefix}${index}/${collection}`
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)